model/src/main/kotlin/OrtResult.kt

```diff
@@ -254,6 +254,18 @@ data class OrtResult(
         return projectsAndPackages
     }
 
+    /*
+     * Return all analyzer errors contained in this [OrtResult].
+     */
+    @JsonIgnore
+    fun getAnalyzerErrors(): List<OrtIssue> = analyzer?.result?.errors?.values.orEmpty().flatten()
+
+    /*
+     * Return all scan errors contained in this [OrtResult].
+     */
+    @JsonIgnore
+    fun getScanErrors(): List<OrtIssue> = scanner?.results?.collectErrors()?.values.orEmpty().flatten()
```

Comment:
It looks like you only use these functions in combination in the next commit, you could already use `collectErrors.values.flatten()` instead without adding new functions.

Reply:
Done.